HelloWorldService/src/main/java/com/gpnk/helloworld/HelloWorldService.java

```diff
@@ -36,6 +38,14 @@ public HelloWorldService() {
      */
     public Optional<WeatherReport> getWeatherForUser(final String userName) {
 
+        if ("foo".equals(userName)) {
```

Comment:
Not sure if we'd want to leave this Exception example or not. The argument for leaving this code is that one can observe "Exception in action".